modules/scripts-utils/dtbook-utils/src/main/resources/xml/dtbook-utils-library.xpl

```diff
@@ -18,5 +18,7 @@
     <p:import href="validate-dtbook/validate-dtbook.xpl"/>
     <p:import href="dtbook-to-mods-meta.xpl"/>
     <p:import href="dtbook-update-links.xpl"/>
+
+    <p:import href="fix-dtbook/fix-dtbook.xpl"/>
```

Comment:
I think I will move the dtbook-cleaner script inside the dtbook-utils module so that

- we don't need to create a new module that we will delete afterwards
- we don't need to expose the new steps to the library yet

For now the dtbook-cleaner script will not be actually exposed as a Pipeline script yet, but I will create a custom release with the script enabled for Save-as-DAISY.

Reply:
Removed from library.